src/mscorlib/corefx/System/Globalization/CultureData.Unix.cs

```diff
@@ -300,5 +307,1772 @@ private static string ConvertIcuTimeFormatString(string icuFormatString)
 
             return StringBuilderCache.GetStringAndRelease(sb);
         }
+        
+        private static string LCIDToLocaleName(int culture)
+        {
+            string cultureName;
+            if (LcidToCultureNameMapper.TryGetValue(culture, out cultureName))
+            {
+                return cultureName;
+            }
+            
+            return null;
+        }
+
+        private static int LocaleNameToLCID(string cultureName)
+        {
+            int lcid;
+            if (CultureNameToLcidMapper.TryGetValue(AnsiToLower(cultureName), out lcid))
+            {
+                return lcid;
+            }
+            
+            return LOCALE_CUSTOM_UNSPECIFIED;
+        }
+
+        private static CultureNameToLcidDictionary CultureNameToLcidMapper
+        {
+            get
+            {
+                if (s_cultureNameToLcidDictionary == null)
+                {
+                    CultureNameToLcidDictionary tempMapping = new CultureNameToLcidDictionary()
+                    {
+                        { "", 0x0007f },
```

Comment:
Actually this can be a good idea as I am expecting to add more data to such table (mapping from culture name to multiple properties). I'll look at  that

---
In reply to: [82091437](https://github.com/dotnet/coreclr/pull/7502#discussion_r82091437) [](ancestors = 82091437)